sketch_map_tool/database/client_flask.py

```diff
@@ -141,7 +141,13 @@ def select_map_frame(uuid: UUID) -> bytes:
     query = "SELECT file FROM map_frame WHERE uuid = %s"
     db_conn = open_connection()
     with db_conn.cursor() as curs:
-        curs.execute(query, [str(uuid)])
+        try:
+            curs.execute(query, [str(uuid)])
+        except psycopg2.errors.UndefinedTable:
+            raise FileNotFoundError_(
+                "In this Sketch Map Tool instance no sketch map has been generated yet. You can only upload sketch maps"
+                " to the instance on which they have been created."
+            )
```

Comment:
This sounds like no sketch maps have ever been created in this instance which is probably not true, but the original sketch map frame (template) is not in the database anymore.
Should we state it more general?
 
> For your uploaded sketch map the original sketch map could not be found. You can only upload sketch maps to the instance on which they have been created.

Reply:
Make sense